src/App.js

```diff
@@ -13,6 +13,7 @@ import SchoolCampaign from './Campaigns/School/SchoolCampaign';
 import ToysCampaign from './Campaigns/Toys/ToysCampaign';
 import MembersForm from './Components/Members/MembersForm';
 import ProjectsForm from './Components/Projects/ProjectsForm';
+import OrganizationDataLayout from './Components/OrganizationData/OrganizationDataLayout/OrganizationDataLayout';
```

Comment:
maybe the folder should be Organization instead of OrganizationData

Reply:
Perfect I change it.